modules/systemd-journal/journal-reader.c

```diff
@@ -234,6 +263,7 @@ __handle_message(JournalReader *self)
 
   journald_foreach_data(self->journal, __handle_data, args);
   __set_message_timestamp(self, msg);
```

Comment:
I know, that the double underscores don't come from your pencil, but if you modify the file, please correct the underscores